packages/common/http/src/client.ts

```diff
@@ -138,6 +139,7 @@ export class HttpClient {
           {[param: string]: string | number | boolean | ReadonlyArray<string|number|boolean>},
     reportProgress?: boolean, responseType: 'arraybuffer',
     withCredentials?: boolean,
+    excludeFromHttpTransfer?: boolean,
```

Comment:
I think it'd be great to use "cache" in the name (or the entire `HttpTransferCache`):
```suggestion
    excludeFromHttpTransferCache?: boolean,
```
May be something like this?
```suggestion
    skipHttpTransferCache?: boolean,
```

Reply:
"skip" is shorter, but "excludeFrom" has an interplay with the "exclude" option from the "withHttpTransferCache" function.